server/model.go

```diff
@@ -12,6 +12,7 @@ import (
 	"os"
 	"slices"
 	"strings"
+	gotmpl "text/template"
```

Comment:
Is there a way we can clean this up or move it to the `github.com/ollama/ollama/template` package to avoid this ugly import?